docs/router-store/api.md

```diff
@@ -1,5 +1,18 @@
 # API
 
+## StoreRouterConnectingModule.forRoot
+
+To connect the Angular router module with the NgRx store module, import the store module via
```

Comment:
Remove the `function` that accepts an object. Its going to be removed from the API

Reply:
Done.